experimental/generation/generator/packages/library/src/mergeAssets.ts

```diff
@@ -154,16 +150,14 @@ export async function mergeAssets(schemaName: string, oldPath: string, newPath:
             await fs.ensureDir(ppath.join(mergedPath, locale))
             feedback(FeedbackType.message, `Create output dir : ${mergedPath} `)
 
-            let oldFileList = []
-            getFiles(oldPath, oldFileList)
-            let newFileList = []
-            getFiles(newPath, newFileList)
+            let oldFileList = await getFiles(oldPath)
```

Comment:
Should the function be getFiles(oldPath, oldFileList) ?

Reply:
I think this is OK.